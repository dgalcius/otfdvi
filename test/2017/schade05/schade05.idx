\indexentry {tensor function!scalar-valued@, scalar-valued|hyperpage}{221}
\indexentry {tensor function!tensor-valued@, tensor-valued|hyperpage}{221}
\indexentry {scalar-valued function|hyperpage}{221}
\indexentry {tensor-valued function|hyperpage}{221}
\indexentry {tensor function|hyperpage}{221}
\indexentry {invariance condition for tensor functions|hyperpage}{221}
\indexentry {tensor functions!invariance condition@, invariance condition for|hyperpage}{221}
\indexentry {representation!of tensor functions|hyperpage}{221}
\indexentry {tensor functions!representation@, representation of|hyperpage}{221}
\indexentry {invariant!tensor functions|hyperpage}{221}
\indexentry {tensor functions! invariant@, invariant|hyperpage}{221}
\indexentry {Cayley-Hamilton theorem!generalized@, generalized|hyperpage}{224}
\indexentry {generalized!Cayley-Hamilton theorem@ Cayley-Hamilton theorem|hyperpage}{224}
\indexentry {Cayley-Hamilton theorem|hyperpage}{224}
\indexentry {invariants|hyperpage}{225}
\indexentry {invariants!polynomial@, polynomial|hyperpage}{225}
\indexentry {irreducible|hyperpage}{225}
\indexentry {reducible|hyperpage}{225}
\indexentry {invariants!reducible@, reducible|hyperpage}{225}
\indexentry {invariants!irreducible@, irreducible|hyperpage}{225}
\indexentry {integrity basis|hyperpage}{225}
\indexentry {syzygy|hyperpage}{225}
\indexentry {invariants!independent@, independent|hyperpage}{226}
\indexentry {simultaneous invariant|hyperpage}{226}
\indexentry {invariants!simultaneous@, simultaneous|hyperpage}{226}
\indexentry {invariants!independent@, independent|hyperpage}{227}
\indexentry {tensor!symmetric@, symmetric|hyperpage}{228}
\indexentry {symmetric!tensor@ (part of a) tensor|hyperpage}{228}
\indexentry {tensor!antimetric@, antimetric|hyperpage}{228}
\indexentry {antimetric!(part of a) tensor|hyperpage}{228}
\indexentry {tensor!orthogonal@, orthogonal|hyperpage}{228}
\indexentry {orthogonal!tensor@ tensor|hyperpage}{228}
\indexentry {invariants!polynomial@, polynomial|hyperpage}{228}
\indexentry {invariant!degree@, degree of an|hyperpage}{229}
\indexentry {invariants!equivalent@, equivalent|hyperpage}{231}
\indexentry {equivalent@equivalent!invariant@ invariants|hyperpage}{231}
\indexentry {tensor!symmetric@, symmetric|hyperpage}{234}
\indexentry {symmetric!tensor@ (part of a) tensor|hyperpage}{234}
\indexentry {tensor!antimetric@, antimetric|hyperpage}{234}
\indexentry {antimetric!(part of a) tensor|hyperpage}{234}
\indexentry {tensor!orthogonal@, orthogonal|hyperpage}{234}
\indexentry {orthogonal!tensor@ tensor|hyperpage}{234}
\indexentry {invariance condition for tensor functions|(hyperpage}{236}
\indexentry {tensor functions!invariance condition@, invariance condition for|(hyperpage}{236}
\indexentry {tensor function|hyperpage}{236}
\indexentry {tensor function!isotropic@, isotropic|hyperpage}{238}
\indexentry {isotropic!tensor function@ tensor function|hyperpage}{238}
\indexentry {invariance condition for tensor functions|)}{238}
\indexentry {tensor functions!invariance condition@, invariance condition for|)}{238}
\indexentry {tensor function!scalar-valued@, scalar-valued|(hyperpage}{238}
\indexentry {scalar-valued function|(hyperpage}{238}
\indexentry {tensor function!scalar-valued@, scalar-valued|)}{238}
\indexentry {scalar-valued function|)}{238}
\indexentry {tensor function!vector-valued@, vector-valued|(hyperpage}{238}
\indexentry {vector-valued function|(hyperpage}{238}
\indexentry {generator|hyperpage}{239}
\indexentry {function basis|hyperpage}{239}
\indexentry {symmetric!tensor@ (part of a) tensor|hyperpage}{239}
\indexentry {tensor!symmetric@, symmetric|hyperpage}{239}
\indexentry {tensor function!vector-valued@, vector-valued|)}{240}
\indexentry {vector-valued function|)}{240}
\indexentry {tensor function!tensor-valued@, tensor-valued|(hyperpage}{240}
\indexentry {tensor-valued function|(hyperpage}{240}
\indexentry {symmetric!tensor@ (part of a) tensor|hyperpage}{241}
\indexentry {tensor!symmetric@, symmetric|hyperpage}{241}
\indexentry {symmetric!tensor@ (part of a) tensor|hyperpage}{242}
\indexentry {tensor!symmetric@, symmetric|hyperpage}{242}
\indexentry {tensor function!tensor-valued@, tensor-valued|)}{243}
\indexentry {tensor-valued function|)}{243}
\indexentry {tensor function!vector-valued@, vector-valued|hyperpage}{243}
\indexentry {tensor function!tensor-valued@, tensor-valued|hyperpage}{243}
\indexentry {vector-valued function|hyperpage}{243}
\indexentry {tensor-valued function|hyperpage}{243}
\indexentry {anisotropy|(hyperpage}{244}
\indexentry {symmetry group|hyperpage}{244}
\indexentry {hemitropic tensor function|hyperpage}{244}
\indexentry {tensor function!hemitropic@, hemitropic|hyperpage}{244}
\indexentry {isotropic!tensor@ tensor|hyperpage}{245}
\indexentry {tensor!isotropic@, isotropic|hyperpage}{245}
\indexentry {tensor!transverse isotropic@, transverse isotropic|hyperpage}{245}
\indexentry {transverse isotropy|hyperpage}{245}
\indexentry {symmetric!tensor@ (part of a) tensor|hyperpage}{246}
\indexentry {tensor!symmetric@, symmetric|hyperpage}{246}
\indexentry {orthogonal!tensor@ tensor|hyperpage}{246}
\indexentry {tensor!orthogonal@, orthogonal|hyperpage}{246}
\indexentry {antimetric!(part of a) tensor|hyperpage}{246}
\indexentry {tensor!antimetric@, antimetric|hyperpage}{246}
\indexentry {transverse isotropy|hyperpage}{247}
\indexentry {anisotropy|)}{248}
