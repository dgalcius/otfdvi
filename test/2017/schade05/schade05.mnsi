<secfile name="Manuscript info">
  <operation>Submit</operation>
  <sec name="SCHADE05">
    <key name="Main file type">PDF</key>
    <key name="Main file name">print/SCHADE05.pdf</key>
    <key name="Stage">200</key>
    <key name="LaTeX stage">\PROOF</key>
    <key name="Publication year">2018</key>
    <key name="First logical page" type="integer">221</key>
    <key name="Last logical page" type="integer">248</key>
    <key name="Page numbering">arabic</key>
    <key name="Page count" type="integer">28</key>
    <sec name="DB info">
      <sec name="PTS Read">
        <key name="Host">db.vtex.vmt</key>
        <key name="Port">3306</key>
        <key name="User">maket</key>
        <key name="Password">D53B58</key>
        <key name="Database">ptsIII</key>
        <key name="Timeout" type="integer">30</key>
      </sec>
      <sec name="PTS Client">
        <key name="Host">www.vtex.vmt</key>
        <key name="Port" type="integer">80</key>
        <key name="End point">/db/xmlrpc/s1.php</key>
        <key name="Post method">ptspost</key>
        <key name="Test method">help</key>
        <key name="Backup directory">y:\db_backup</key>
        <key name="SSL enabled" type="boolean">No</key>
      </sec>
      <sec name="PTS Info">
        <key name="PTS Info Address">https://pts.vtex.vmt/db/vtexpts/item/check/oper</key>
        <key name="VTEXPTS MS info URL">https://pts.vtex.vmt/db/vtexpts/api/item/%s/info</key>
        <key name="Elsevier journal stylesheets address"></key>
        <key name="PTSIII info URL"></key>
        <key name="Project info URL"></key>
        <key name="VTEXPTS worker info URL">https://pts.vtex.vmt/db/vtexpts/worker/info?uid=%s</key>
      </sec>
    </sec>
    <sec name="File info">
      <key name="Publisher">DGBOOK</key>
      <key name="Project">SCHADE</key>
      <key name="Manuscript">SCHADE05</key>
      <key name="Stage">202</key>
      <key name="MS position"></key>
      <key name="Computer name">1GSRED478</key>
      <key name="User name">juraten</key>
      <key name="TID">juraten</key>
      <key name="Pages"></key>
      <key name="Format">2016</key>
      <key name="Distribution">vtex</key>
      <key name="PDF type">print</key>
      <key name="LaTeX file name">schade05.tex</key>
      <key name="Destination">PDF</key>
      <key name="ISSN/ISBN">978-3-11-040425-8</key>
      <key name="Opened from">local</key>
      <key name="PDF.Maker">pdfwf_luatex</key>
      <key name="Spelled"></key>
      <key name="Graphics directory"></key>
      <key name="Copy to DB" type="boolean">Yes</key>
      <key name="Compiler cmd line"></key>
      <key name="VTEXPTS stage">E100</key>
      <sec name="History" type="text">
        2018.05.31 09:38
      </sec>
    </sec>
    <sec name="LaTeX info">
      <key name="Publisher">DGBook</key>
      <key name="Project">SCHADE</key>
      <key name="Version"></key>
      <key name="DocType"></key>
      <key name="Format">2016</key>
      <key name="Distribution">vtex</key>
      <key name="Destination">PDF</key>
      <key name="DVI.Maker">luadvi</key>
      <key name="DVI.Viewer">xdviwindo</key>
      <key name="PS.Maker">xdvipsk</key>
      <key name="PS.Viewer">GSView</key>
      <key name="PDF.Maker">pdfwf_luatex</key>
      <key name="PDF.Viewer">Acrobat</key>
      <key name="Run before Open">fhgextract,pdfwf_images (full),VTEXPTS2LaTeX,book_aux_toc,Open Source PDF allways</key>
      <key name="Run before Make"></key>
      <key name="Run after Make">FlsUpdated</key>
      <key name="Run before Send">pdfwf_images (full),pdfwf_compile_before_send,pdfwf_mszip,kt-ltx-chk-auto</key>
      <key name="Run after Send"></key>
      <key name="Do not copy from server"></key>
      <key name="Send to server">tex,pdf,log,aux,data,out,pgrefs,*.specials,*.pdf_print_me,*.opentype.map,*.encodings.map</key>
      <key name="Template">schade.for</key>
      <key name="Type page width" type="float">170</key>
      <key name="Type page height" type="float">240</key>
      <key name="Text area width" type="float">122</key>
      <key name="Text area height" type="float">193</key>
      <key name="Column width" type="float">122</key>
      <key name="Run after confirm send">pdfwf_fhg,pdfwf_printpdf</key>
      <key name="Main file type">PDF</key>
      <key name="Track changes" type="boolean">Yes</key>
      <key name="Update pubonline date" type="boolean">No</key>
      <key name="UTF-8" type="boolean">Yes</key>
      <key name="PDF type">print</key>
      <key name="Main file name">print/&amp;ms;.pdf</key>
      <key name="MS subdirs">www,print</key>
      <key name="Do not Send subdirs">images</key>
      <key name="Update doctopic" type="boolean">No</key>
      <key name="GCNS" type="boolean">No</key>
      <key name="GCNS start date"></key>
      <key name="Prelim templates">FM,BM</key>
      <key name="Differ.DVI"></key>
      <key name="Differ.DVI.Meta.Default"></key>
      <key name="Differ.DVI.Meta.Variable"></key>
    </sec>
  </sec>
</secfile>