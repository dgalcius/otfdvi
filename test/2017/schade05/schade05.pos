\relax 
\vtexsavepos@pdf {table0}{23091796sp}{30834826sp}{236}
\vtexsavepos@pdf {table0}{3356424sp}{19509364sp}{243}
\vtexsavepos@pdf {table0}{5594051sp}{7887388sp}{246}
\vtexsavepos@pdf {table0after}{5594051sp}{6683233sp}{246}
