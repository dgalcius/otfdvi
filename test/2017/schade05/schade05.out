\BOOKMARK [0][-]{schade05.chapter.5:1}{\376\377\0005\000\040\000R\000e\000p\000r\000e\000s\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000T\000e\000n\000s\000o\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{}% 1
\BOOKMARK [1][-]{schade05.section.1:2}{\376\377\0005\000.\0001\000\040\000b\000l\000a\000c\000k\000T\000h\000e\000\040\000B\000a\000s\000i\000c\000\040\000I\000d\000e\000a\000\040\000o\000f\000\040\000t\000h\000e\000\040\000R\000e\000p\000r\000e\000s\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000T\000e\000n\000s\000o\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{schade05.chapter.5:1}% 2
\BOOKMARK [1][-]{schade05.section.2:5}{\376\377\0005\000.\0002\000\040\000b\000l\000a\000c\000k\000G\000e\000n\000e\000r\000a\000l\000i\000z\000e\000d\000\040\000C\000a\000y\000l\000e\000y\040\023\000H\000a\000m\000i\000l\000t\000o\000n\000\040\000T\000h\000e\000o\000r\000e\000m}{schade05.chapter.5:1}% 3
\BOOKMARK [1][-]{schade05.section.3:9}{\376\377\0005\000.\0003\000\040\000b\000l\000a\000c\000k\000I\000n\000v\000a\000r\000i\000a\000n\000t\000s\000\040\000o\000f\000\040\000V\000e\000c\000t\000o\000r\000s\000\040\000a\000n\000d\000\040\000S\000e\000c\000o\000n\000d\000-\000O\000r\000d\000e\000r\000\040\000T\000e\000n\000s\000o\000r\000s}{schade05.chapter.5:1}% 4
\BOOKMARK [2][-]{schade05.subsection.1:12}{\376\377\0005\000.\0003\000.\0001\000\040\000b\000l\000a\000c\000k\000I\000n\000v\000a\000r\000i\000a\000n\000t\000s\000\040\000o\000f\000\040\000V\000e\000c\000t\000o\000r\000s}{schade05.section.3:9}% 5
\BOOKMARK [2][-]{schade05.subsection.2:13}{\376\377\0005\000.\0003\000.\0002\000\040\000b\000l\000a\000c\000k\000I\000n\000d\000e\000p\000e\000n\000d\000e\000n\000t\000\040\000I\000n\000v\000a\000r\000i\000a\000n\000t\000s\000\040\000o\000f\000\040\000a\000\040\000S\000e\000c\000o\000n\000d\000-\000O\000r\000d\000e\000r\000\040\000T\000e\000n\000s\000o\000r}{schade05.section.3:9}% 6
\BOOKMARK [2][-]{schade05.subsection.3:15}{\376\377\0005\000.\0003\000.\0003\000\040\000b\000l\000a\000c\000k\000I\000r\000r\000e\000d\000u\000c\000i\000b\000l\000e\000\040\000I\000n\000v\000a\000r\000i\000a\000n\000t\000s\000\040\000o\000f\000\040\000S\000e\000c\000o\000n\000d\000-\000O\000r\000d\000e\000r\000\040\000T\000e\000n\000s\000o\000r\000s}{schade05.section.3:9}% 7
\BOOKMARK [2][-]{schade05.subsection.4:35}{\376\377\0005\000.\0003\000.\0004\000\040\000S\000u\000m\000m\000a\000r\000y}{schade05.section.3:9}% 8
\BOOKMARK [1][-]{schade05.section.4:37}{\376\377\0005\000.\0004\000\040\000b\000l\000a\000c\000k\000I\000s\000o\000t\000r\000o\000p\000i\000c\000\040\000T\000e\000n\000s\000o\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{schade05.chapter.5:1}% 9
\BOOKMARK [2][-]{schade05.subsection.1:38}{\376\377\0005\000.\0004\000.\0001\000\040\000b\000l\000a\000c\000k\000I\000n\000v\000a\000r\000i\000a\000n\000c\000e\000\040\000C\000o\000n\000d\000i\000t\000i\000o\000n\000s}{schade05.section.4:37}% 10
\BOOKMARK [2][-]{schade05.subsection.2:42}{\376\377\0005\000.\0004\000.\0002\000\040\000b\000l\000a\000c\000k\000S\000c\000a\000l\000a\000r\000-\000V\000a\000l\000u\000e\000d\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{schade05.section.4:37}% 11
\BOOKMARK [2][-]{schade05.subsection.3:44}{\376\377\0005\000.\0004\000.\0003\000\040\000b\000l\000a\000c\000k\000V\000e\000c\000t\000o\000r\000-\000V\000a\000l\000u\000e\000d\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{schade05.section.4:37}% 12
\BOOKMARK [2][-]{schade05.subsection.4:48}{\376\377\0005\000.\0004\000.\0004\000\040\000b\000l\000a\000c\000k\000T\000e\000n\000s\000o\000r\000-\000V\000a\000l\000u\000e\000d\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{schade05.section.4:37}% 13
\BOOKMARK [2][-]{schade05.subsection.5:51}{\376\377\0005\000.\0004\000.\0005\000\040\000b\000l\000a\000c\000k\000S\000u\000m\000m\000a\000r\000y}{schade05.section.4:37}% 14
\BOOKMARK [1][-]{schade05.section.5:53}{\376\377\0005\000.\0005\000\040\000b\000l\000a\000c\000k\000C\000o\000n\000s\000i\000d\000e\000r\000i\000n\000g\000\040\000A\000n\000i\000s\000o\000t\000r\000o\000p\000y}{schade05.chapter.5:1}% 15
